pkg/logbridge/logbridge_test.go

```diff
@@ -0,0 +1,161 @@
+package logbridge
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/square/p2/pkg/logging"
+)
+
+const newLine = byte(10)
+
+type TrackingWriter struct {
+	bytes     []byte
+	numWrites int
+}
+
+func (tw *TrackingWriter) Write(p []byte) (int, error) {
+	tw.numWrites++
+	return len(p), nil
```

Comment:
this function doesn't appear to actually update the byte array inside the writer, intentional?